api/api/src/main/scala/uk/ac/wellcome/platform/api/Router.scala

```diff
@@ -246,5 +288,10 @@ class Router(elasticClient: ElasticClient,
   lazy val worksService =
     new WorksService(new ElasticsearchService(elasticClient))
 
+  lazy val collectionService =
+    new CollectionService(elasticClient)
+
+  lazy val logger = Logger(this.getClass.getName)
```

Comment:
The way we normally add logging is to extend from `grizzled.sl4j.Logging`; is there a reason you didn't do that here?

Reply:
`error` is defined already is this class